simulators/rpc-compat/src/main.rs

```diff
@@ -144,3 +162,36 @@ dyn_async! {
         }
     }
 }
+
+dyn_async! {
+    async fn test_history_local_content_content_present<'a>(test: &'a mut Test, client: Option<Client>) {
```

Comment:
```suggestion
    async fn test_history_local_content_expect_content_present<'a>(test: &'a mut Test, client: Option<Client>) {
```
would be more consistent with the naming

Reply:
resolved